hasura/migrations/1575910000000_init/up.sql

```diff
@@ -23,66 +32,34 @@ CREATE TABLE "Player" (
   "id" uuid DEFAULT public.gen_random_uuid() NOT NULL,
   "username" text NOT NULL,
   "totalXp" numeric DEFAULT 0,
-  "rank" player_rank NOT NULL DEFAULT 'PLAYER',
-  "links" json,
-  "sentences" json
+  "role" text,
+  "timezone" int,
+  "enneagram" enneagram_type,
+  "skills" text[]
```

Comment:
I think we should have a table of skills: https://hasura.io/docs/1.0/graphql/manual/schema/enums.html#using-foreign-key-references-to-a-single-column-table

That way we can allow people to search/filter based on the skills of a user and have autocomplete for a standard list of skills but also allow users to define new ones and insert them into the Skills table

Reply:
Having the skills in a table would indeed allow to search/filter more easily but letting users insert new ones can lead to fake doubles, like mispelling, or calling the same thing with different names or punctuation. 
We should then have an autocomplete option like with algolia, and at some point we we'll have to aggregate same skills with different names.

Reply:
Yea, we can prepopulate with a standard list of skills to start and them add custom skills feature later to keep it simple

Reply:
It seems that hasura's enums are not the good solution for skills for different reasons:
- PostgreSQL can't have columns with an array of reference to another table, which would mean people would only have one skill (https://stackoverflow.com/questions/41054507/postgresql-array-of-elements-that-each-are-a-foreign-key)
- In hasura, the elements of the enums are generated in the metadata, which means we have to manually regenerate it everytime the list of skills is updated (https://hasura.io/docs/1.0/graphql/manual/schema/enums.html#current-limitations)

The only solution I see here is to have a Skill table with a many-to-many relationships between Player-Skill